components/navbar/Navbar.js

```diff
@@ -242,7 +242,7 @@ export default function Navbar() {
           </div>
           <div className="pt-4 pb-3 border-t border-primary-medium">
             <div className="flex items-center px-5">
-              <div className="flex items-center md:ml-6">
+              <div className="flex items-center md:ml-6 flex-wrap">
```

Comment:
Maybe you can center this? Centered items looks a bit nicer, imo.
![image](https://github.com/EddieHubCommunity/LinkFree/assets/108367944/2523eefe-885c-4c49-885f-7b6278b47046)